### PR TITLE
docs(changelog): add breaking change to DynamicComponentLoader to changelog of beta 16

### PR DESCRIPTION
**Pull request by ChrisPearce**
- **Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
  Adds missing breaking changes relating to DynamicComponentLoader and AppViewManager to Beta 16 change-log.
- **What is the current behavior?** (You can also link to an open issue here)
  Beta 16 change-log is missing breaking changes for refactoring of DynamicComponentLoader and AppViewManager.
- **What is the new behavior (if this is a feature change)?**
  Beta 16 change-log has been updated to include breaking changes for refactoring of DynamicComponentLoader and AppViewManager
- **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
  No
- **Other information**:
  Text copied from commit message for efbd446d18e6e0380beafcad6e94a7751d788623 by @tbosch

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -74,6 +74,12 @@ fakeAsync(inject([...], (...) => {...}))
 
 * - pipes now take a variable number of arguments, and not an array that contains all arguments.
 
+* `DynamicComponentLoader.loadIntoLocation` has been removed. Use `@ViewChild(‘myVar’, read: ViewContainerRef)` to get hold of a `ViewContainerRef` at an element with variable `myVar`.
```
Comment (zoechi) [inline CHANGELOG.md]: How can `ViewContainerRef` be used as replacement for `DynamicComponentLoader.loadIntoLocation`?
"Use `@ViewChild(...)` seems to be related to the following `DynamicComponentLoader.loadNextToLocation`

Comment (zoechi) [inline CHANGELOG.md]: `@ViewChild(‘myVar’, read: ViewContainerRef)` should be  
`@ViewChild(‘myVar’, {read: ViewContainerRef})`

**Comment by rkirov** [comment]
Was this a failure of the changelog gen, or those changes were not documented in the original PR?

**Comment by ericmartinezr** [comment]
@rkirov there's the commit message https://github.com/angular/angular/pull/8124/commits/efbd446d18e6e0380beafcad6e94a7751d788623 looks like it wasn't picked up

**Comment by rkirov** [comment]
My guess is that `refactor(...)` is being skipped by the changelog generator. LGTM

**Comment by rkirov** [comment]
superceded by https://github.com/angular/angular/pull/8262

**Pull request closed without merging**